test/test-applications/regression/Directory.Build.props

```diff
@@ -1,3 +0,0 @@
-<Project>
-  <Import Project="..\Directory.Build.props" />
-</Project>
```

Comment:
Can you delete these references from the sln files too - there's a few "missing files" in solution explorer since the last cleanup too

Reply:
oops, thanks!

Reply:
Fixed in e9a092c7ec1846561ccc80477d8268027a030cf9